### ISSUE TITLE
capture hash UMI counts for non polyA hashes

### ISSUE THREAD

**Issue by eliza-barkan** [bbi-lab/bbi-sci#27]
Would it be possible to capture hashes that do not have a polyA tail (i.e. some specific sequence instead)?

Thanks!

**Comment by hpliner** [comment]
Currently the script looks for AAAA in positions 11:15 of the read. Would the alternative be something similar? I.e. a sequence in a specific position?

**Comment by eliza-barkan** [comment]
Yes, maybe there could be two options where the default is polyA or you can input a custom sequence and specify at which positions to look for the sequence at
